### ISSUE TITLE
Avatar added menu

### ISSUE THREAD

**Issue by theboom1** [Frooxius/NeosPublic#1119]
After you create an avatar we should have 3 option to cancel, equipt avatar, the third should be edit mode and all the material orbs should all come out with 3 main features one to swap main texture second to change the main color third part is selecting the shader for the orb then drop down list of the built in shaders. This is the main thing most user will common users will want quick access without using the material gun. But to access the full access then the material gun.

**Comment by Frooxius** [comment]
I'm sorry, but this option won't be added to the context menu, it doesn't really belong there for editing avatar and editing should generally use the actual editing tools.

It's unfortunately not possible to do the editing as you propose it, since the avatars come in huge variety of configurations, with different materials, sets of textures and so on.

Better thing would be to see what are the underlying issues? We can change the avatar creator or material gun to make it easier to use and access things quicker.

**Comment by theboom1** [comment]
Then give the material gun the option to draw all material orbs.

**Comment by H3BO3** [comment]
That's already possible by grabbing a ref from the model to the avatar root and hitting secondary.

**Comment by TehTurk** [comment]
Yup. But also to add from discussions I've had with Boom, they want something that's more akin to an easy avatar editor with Tools, but the issue right now is there's nothing in game to hook into the SkinnedMeshRenderer material list. Material Gun can already spawn all materials and you can edit them, but that's the only way atm. So this probably goes more along the lines of collections.

**Comment by Frooxius** [comment]
One thing I plan on changing on the material gun is so extracting material from a skinned mesh will extract all of them, rather than just one. This should make the access to them much easier and won't require an inspector.

**Comment by TehTurk** [comment]
Wanted to add to this issue, if this type of behavior is added can I suggest spawning the SkinnedMeshRenderer WorkerPanel instead? Managing Copys, or Reassigning the right References become alot easier, or alot smoother with this in mind and also saves on World Clutter.
